policyengine_us/variables/gov/states/co/tax/income/subtractions/pension/co_social_security_subtraction_indv.py

```diff
@@ -0,0 +1,67 @@
+from policyengine_us.model_api import *
+
+
+class co_social_security_subtraction_indv(Variable):
+    value_type = float
+    entity = Person
+    label = "Colorado social security subtraction for eligible individuals"
+    defined_for = StateCode.CO
+    unit = USD
+    reference = (
+        "https://tax.colorado.gov/sites/tax/files/documents/DR0104AD_2022.pdf#page=1",
+        "https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=12",
+        "https://law.justia.com/codes/colorado/2022/title-39/article-22/part-1/section-39-22-104/"
+        # C.R.S. 39-22-104(4)(g)(III)
+    )
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.co.tax.income.subtractions.pension
+        if p.social_security_subtraction_available:
```

Comment:
I'd suggest switching these so you can un-indent the main code
```suggestion
        if not p.social_security_subtraction_available:
            return 0
```